project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/logging/Loggers.kt

```diff
@@ -4,6 +4,5 @@ import mu.KotlinLogging
 
 object Loggers {
     val statLogger = KotlinLogging.logger("STATISTICS")
-    val workingLogger = KotlinLogging.logger("WORKING")
-    val generalLogger = KotlinLogging.logger("GENERAL")
+    val generalLogger = KotlinLogging.logger {}
```

Comment:
It should be called in the class where we are logging something because the name of this logger is formed from the call site. In your version, generalLogger will always have the name "org.plan.research.minimization.plugin.logging.Loggers".

Like we have the class `MinimizationStageExecutorService` and add to its fields something like `private val logger = KotlinLogging.logger {}`. It leads that this `logger` will have the name "org.plan.research.minimization.plugin.services.MinimizationStageExecutorService". So, when we use this `logger` to log something, its message will be more informative because it will automatically say the call site of this logger and where to search its message.